plugin/include/txn_box/common.h

```diff
@@ -32,21 +37,6 @@ namespace YAML
 class Node;
 }
 
-/// Generate an @c Errata from a format string and arguments.
-template <typename... Args>
-swoc::Errata
-Error(std::string_view const &fmt, Args &&... args)
-{
-  return std::move(swoc::Errata().note_v(swoc::Severity::ERROR, fmt, std::forward_as_tuple(args...)));
-}
-
-template <typename... Args>
-swoc::Errata
-Warning(std::string_view const &fmt, Args &&... args)
-{
-  return std::move(swoc::Errata().note_v(swoc::Severity::WARN, fmt, std::forward_as_tuple(args...)));
-}
-
```

Comment:
Why removing it? I think this was a nice way of expressing the error/warn,etc
```
template <typename... Args>
swoc::Errata
Error(std::string_view const &fmt, Args &&... args)
{
  return swoc::Errata(S_ERROR, fmt, std::forward_as_tuple(args...)));
}
```

Reply:
It's just a severity code to be passed in now, I didn't think it was worth it.